pkg/kclient/kclient.go

```diff
@@ -21,6 +24,7 @@ type Client struct {
 	KubeConfig       clientcmd.ClientConfig
 	KubeClientConfig *rest.Config
 	Namespace        string
+	OperatorClient   *operatorsclientset.OperatorsV1alpha1Client
```

Comment:
I could be completely off base here but shouldn’t we use v1 version of the client https://github.com/operator-framework/operator-lifecycle-manager/tree/release-4.1/pkg/api/client/clientset/versioned/typed/operators/v1

Reply:
We're using this because `v1` doesn't have the `ClusterServiceVersionList` type defined or method available to fetch these info. I looked for it but couldn't find one in v1.

Reply:
Wouldn’t v1alpha1 soon deprecated after a while?

Reply:
I think we're stepping into discussion that can be better had by the OLM folks. We (as in the odo team folks) can just speculate. :wink: 

I'll open an issue there asking for the availability of stuff we need in `v1`.

Reply:
https://github.com/operator-framework/operator-lifecycle-manager/issues/1317

Reply:
After opening the issue, I looked around a bit. This seems to be a `release-4.1` thing. In the latest `master`, I see CSV related structs and methods outside the `v1alpha1`. So maybe we can use the `v1` APIs of OLM when we change our dependency to something newer than `release-4.1`